tests/test-api.php

```diff
@@ -46,15 +44,15 @@ function test_api_put() {
 	}
 
 	function test_api_delete() {
-		SP_API()->put( SP_API()->get_doc_type() . '/123456', '{"post_id":123456}' );
+		SP_API()->put( SP_API()->get_doc_type() . '/223456', '{"post_id":223456}' );
```

Comment:
Why were these IDs changed from 123456 to 223456?

Reply:
short version: each test used to be isolated from the others and now they aren't. it's a tax I'm willing to pay for the performance gains. in moving from per-test databases and indexes to per-class databases and indexes, I introduced data leaks in some places; this was one. On line 38, we're `PUT`ing `/123456`, so if this line `PUT`s it again it will produce a `200` vs a `201` response, failing the next assertion. by using a different made-up number, we avoid that collision.